test/powershell/engine/Encoding.Tests.ps1

```diff
@@ -0,0 +1,225 @@
+Describe "Encoding classes and methods are available" -Tag CI {
+    BeforeAll {
+        $testString = "t" + ([char]233) + "st"
+        $provider = get-item $TESTDRIVE
+        $testFile = "${TESTDRIVE}/file.txt"
+        $preamble = @{
+            Ascii = ''
+            BigEndianUTF32 = '254-255'
+            BigEndianUnicode = '254-255'
+            Byte = '255-254'
+            Default = ''
+            Oem = ''
+            String = '255-254'
+            UTF32 = '255-254-0-0'
+            UTF7 = ''
```

Comment:
I understand (maybe the api designer didn't think it was worth returning an array of possible preambles, and the implementer decided it was better to return nothing rather than choosing one possible preamble.)

Regardless, it's worth a comment on why utf7 is missing an expected bom, after all, we aren't calling it UTF7NoBOM.